### PR TITLE
Model updates 20210731

### PR DESCRIPTION
**Pull request by invisig0th**

### CODE REVIEW

File: synapse/models/orgs.py [hunk 1]
```diff
@@ -10,6 +10,17 @@
...
 
+dealstatus = (
+    (0, 'new'),
+    (0, 'research'),
+    (10, 'pitching'),
+    (20, 'offer made'),
+    (50, 'negotiation'),
+    (50, 'committed'),
+    (60, 'delivered'),
+    (70, 'done'),
```
Comment (vEpiphyte) [inline synapse/models/orgs.py:21]: is stalled or cancelled/dead also a valid status?

Comment (invisig0th) [reply synapse/models/orgs.py:21]: Yea, def need to talk through these defaults with shevenom and maybe some outsiders :D

File: synapse/models/biz.py [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+                        'doc': 'The date/time that the RFP was posted.',
+                    }),
+                    ('quesdue', ('time', {}), {
+                        'doc': 'The date/time that questions are due.',
+                    }),
+                    ('propdue', ('time', {}), {
+                        'doc': 'The date/time that proposals are due.',
+                    }),
+                    ('contact', ('ps:contact', {}), {
+                        'doc': 'The contact information given for the org reqesting offers.',
```
Comment (reign-git) [inline synapse/models/biz.py:68]: Change "reqesting" to "requesting"

File: synapse/models/person.py [hunk 3]
```diff
@@ -47,8 +47,46 @@ def getModelDefs(self):
...
+                    ('org', ('ou:org', {}), {
+                        'doc': 'The org that this work history orgname refers to.',
+                    }),
+                    ('orgname', ('ou:name', {}), {
+                        'doc': 'The reported name of the org the contact worked for.',
+                    }),
+                    ('orgfqdn', ('inet:fqdn', {}), {
+                        'doc': 'The reported fqdn of the org the contact worked for.',
+                    }),
+                    ('jobtype', ('ou:jobtype', {}), {
```
Comment (mbvertex) [inline synapse/models/person.py:68]: Would this include type as in whether the role is PT/FTE/Contractor? (Also, would we want to be able to tie someone's work history to a specific contract?

File: synapse/models/biz.py [hunk 4]
```diff
@@ -0,0 +1,204 @@
...
+                    # TODO ('upc', ('biz:upc', {}), {}),
+                    ('summary', ('str', {}), {
+                        'doc': 'A brief summary of the product.',
+                        'disp': {'hint': 'text'},
+                    }),
+                    ('price:retail', ('econ:price', {}), {
+                        'doc': 'The MSRP price of the product.',
+                    }),
+                    ('price:bottom', ('econ:price', {}), {
+                        'doc': 'The minium offered or observed price of the product.',
```
Comment (reign-git) [inline synapse/models/biz.py:159]: Change "minium" to "minimum"

File: synapse/models/biz.py [hunk 5]
```diff
@@ -0,0 +1,204 @@
...
+                        'doc': 'An arbitrary name for this stake. Can be non-contact like "pool".',
+                    }),
+                    ('asof', ('time', {}), {
+                        'doc': 'The time the stake is being measured. Likely as part of an ou:vitals.',
+                    }),
+                    ('shares', ('int', {}), {
+                        'doc': 'The number of shares represented by the stake.',
+                    }),
+                    ('invested', ('econ:price', {}), {
+                        'doc': 'The emount of money invested in the captable iteration.',
```
Comment (reign-git) [inline synapse/models/biz.py:188]: Change "emount" to "amount"

Comment (reign-git) [inline synapse/models/biz.py:188]: Should captable be "cap table"? Just checking

File: synapse/models/biz.py [hunk 6]
```diff
@@ -0,0 +1,204 @@
+import synapse.lib.module as s_module
+
+'''
+Model elements related to sales / bizdev / procurement
+'''
+
+class BizModule(s_module.CoreModule):
+    def getModelDefs(self):
+        modl = {
+            'types': (
```
Comment (reign-git) [inline synapse/models/biz.py:10]: Do we want to add a biz:proposal form (or something of the like) to capture rfp proposals?

File: synapse/models/biz.py [hunk 7]
```diff
@@ -0,0 +1,204 @@
...
+                ('biz:prodtype', ('taxonomy', {}), {
+                    'doc': 'A product type taxonomy.',
+                    'interfaces': ('taxonomy',),
+                }),
+            ),
+            'forms': (
+                ('biz:dealtype', {}, ()),
+                ('biz:prodtype', {}, ()),
+                ('biz:dealstatus', {}, ()),
+                ('biz:rfp', {}, (
```
Comment (reign-git) [inline synapse/models/biz.py:40]: Do we want to add a :proposal prop to capture  biz:proposal nodes that were submitted in response to the RFP?

Comment (reign-git) [inline synapse/models/biz.py:40]: Do we want to add a :file prop to link the RFP file bytes to the biz:rfp node?

Comment (reign-git) [inline synapse/models/biz.py:40]: Do we want add a :commencement property to capture the expected rfp commencement date?

File: synapse/models/biz.py [hunk 8]
```diff
@@ -0,0 +1,204 @@
...
+                    }),
+                    ('contact', ('ps:contact', {}), {
+                        'doc': 'The contact information given for the org reqesting offers.',
+                    }),
+                    ('purchases', ('econ:purchase', {}), {
+                        'doc': 'Any known purchases that resulted from the RFP.',
+                    }),
+                    ('requirements', ('array', {'type': 'ou:goal'}), {}),
+                )),
+                ('biz:deal', {}, (
```
Comment (reign-git) [inline synapse/models/biz.py:75]: Curious of the diff b/t biz:deal and ou:contract? And why we opted to created this form vs extending the contract node?

File: synapse/models/biz.py (hunk 7 above)
Comment (reign-git) [inline synapse/models/biz.py:40]: Do we want to add a :vendors prop (type ps:contact) to capture invited vendors?

File: synapse/models/biz.py (hunk 6 above)
Comment (reign-git) [inline synapse/models/biz.py:10]: Do we want to add a biz:rfp:proposal:score type to capture proposal scores?

File: synapse/models/biz.py (hunk 2 above)
Comment (reign-git) [inline synapse/models/biz.py:68]: Change "reqesting" to "requesting"

File: synapse/models/biz.py (hunk 6 above)
Comment (invisig0th) [reply synapse/models/biz.py:10]: Eventually yea, but i left it TBD on purpose for now ( let it come back up later )

Comment (invisig0th) [reply synapse/models/biz.py:10]: This ends up being TBD for now as well because it would go on a proposal node

File: synapse/models/biz.py (hunk 7 above)
Comment (csimsv2611) [reply synapse/models/biz.py:40]: I agree with Reign. I think a `:vendors` prop would work well here

Comment (invisig0th) [reply synapse/models/biz.py:40]: Experience tells me this might be different ( per-proposal ) for a multi-prop award so TBD as well

Comment (invisig0th) [reply synapse/models/biz.py:40]: Is it common to have an invite-only RFP ( or a published set of specifically invited vendors? )

File: synapse/models/biz.py (hunk 8 above)
Comment (invisig0th) [reply synapse/models/biz.py:75]: Happy to discuss, but it's a classic "potential vs actual" model issue, as well as a different set of tracking characteristics.

File: synapse/models/biz.py (hunk 5 above)
Comment (invisig0th) [reply synapse/models/biz.py:188]: Good question, i'll check. :D

File: synapse/models/person.py (hunk 3 above)
Comment (invisig0th) [reply synapse/models/person.py:68]: Designating the taxonomy hierarchy would be up to the users and power-up developers, but my thoughts were more about the type of work being done. I think we could probably add an enum about full-time vs part-time to deconflate that dimension.

Re contracts: Is this in regards to their employment contract or more like a gov "i worked on the foo contract" type thing?

Comment (mbvertex) [reply synapse/models/person.py:68]: I was thinking more of the latter scenario for contracts (like if we were modeling Reign at VTX and wanted to show that she worked on wm and pg, how could we do that?)

File: synapse/models/biz.py (hunk 7 above)
Comment (invisig0th) [reply synapse/models/biz.py:40]: This would be an `:rfp` secondary prop on the proposal node so remains TBD

File: synapse/models/orgs.py [hunk 9]
```diff
@@ -216,7 +343,9 @@ def getModelDefs(self):
                         'doc': 'Set if a goal has a limited time window.',
                     }),
                 )),
+                ('ou:camptype', {}, ()),
                 ('ou:campaign', {}, (
+                    # political campaign, funding round, ad campaign, fund raising
```
Comment (vEpiphyte) [inline synapse/models/orgs.py:348]: This comment doesn't exclude the use of ou:campaign for the risk:attack:campaign property but does confuse it a little bit with the stated description of `Represents an orgs activity in pursuit of a goal.`

**Review by mikemoritz — APPROVED** [review]

**Pull request merged**